modules/mod/kickban.rb

```diff
@@ -1,70 +1,87 @@
+# frozen_string_literal: true
+
 # Copyright Erisa A. (erisa.moe) 2016-2020
 
 module YuukiBot
   module Mod
-
-    $cbot.add_command(:kick,
+    YuukiBot.crb.add_command(
```

Comment:
The mod commands need to have checks for whether the user using the command actually has permission to perform the action on the target user.